acousticbrainz/views/datasets.py

```diff
@@ -21,6 +21,14 @@ def view(id):
     )
 
 
+@datasets_bp.route("/<uuid:id>/json")
+def view_json(id):
+    ds = dataset.get(id)
```

Comment:
OK. Fine for merge now then but this should be quite high in the list of
next things

On Fri, Jun 12, 2015 at 11:06 AM, Roman Tsukanov <notifications@github.com>
wrote:

> In acousticbrainz/views/datasets.py
> <https://github.com/metabrainz/acousticbrainz-server/pull/102#discussion_r32300438>
> :
>
> > @@ -21,6 +21,14 @@ def view(id):
> >      )
> >
> >
> > +@datasets_bp.route("/<uuid:id>/json")
> > +def view_json(id):
> > +    ds = dataset.get(id)
>
> We didn't implement privacy features for datasets yet.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metabrainz/acousticbrainz-server/pull/102/files#r32300438>
> .
>